src/data/fintype/basic.lean

```diff
@@ -1390,6 +1390,59 @@ lemma finset.univ_map_embedding {α : Type*} [fintype α] (e : α ↪ α) :
   univ.map e = univ :=
 by rw [←e.equiv_of_fintype_self_embedding_to_embedding, univ_map_equiv_to_embedding]
 
+/-- There exists a bijection between two types of the same finite cardinality, which extends a given
+bijection on a given finset. -/
+lemma finset.exists_equiv_extend_of_card_eq [fintype α] [decidable_eq β] {t : finset β}
+  (hαt : fintype.card α = t.card) {s : finset α} :
+  ∀ {f : α → β} (hfst : s.image f ⊆ t) (hfs : set.inj_on f s),
```

Comment:
This might or might not be more idiomatic.
```suggestion
  ∀ {f : α → β} (hfst : set.maps_to f s t) (hfs : set.inj_on f s),
```
and won't require `[decidable_eq β]`

Reply:
Actually, it seems I didn't need `[decidable_eq β]` anyway.  Thanks for the catch!

I like the suggested version less because it introduces the complication of switching between set and finset inclusion.  Someone could define a `finset.maps_to` for this purpose ...